browser/index.mjs

```diff
@@ -1,29 +1,33 @@
 // @ts-check
 
 import { predicate, parse } from './queryan.js'
-import { call, string, uniq } from './helpers.js'
+import { call, string } from './helpers.js'
 import search from './search.mjs'
 import issuesFn from './issues.mjs'
 import state from './state.mjs'
 
 const issues = Array.from(document.querySelectorAll('.straw-issues article'))
 const issueElementsByToken = call(() => {
   const directoryTokenRegex = /\w+(?=\/)/g
-  const otherTokensRegex = /[@#]\S+/g
 
   return issues.reduce((acc, elem) => {
     const bookmark = elem.querySelector('.straw-bookmark')
 
-    uniq(bookmark.textContent.match(directoryTokenRegex)).forEach((token) => {
-      token = `/${token}`
+    if (bookmark?.textContent == null) return {};
 
-      acc[token] = acc[token] || []
-      acc[token].push(elem)
+    (bookmark.textContent.match(directoryTokenRegex) || []).forEach((token) => {
+      token = `/${token}`
+      acc[token] ??= []
+      if (!acc[token].includes(token)) acc[token].push(elem)
     })
 
-    uniq(elem.textContent.match(otherTokensRegex)).forEach((token) => {
-      acc[token] = acc[token] || []
-      acc[token].push(elem)
+    Array.from(
+      elem.querySelectorAll('.straw-mention, .straw-hashtag') || []
```

Comment:
Just query the tokens that are wrapped in an anchor with a useful class name. This removes the need for maintaining another regex query in the JS code.